src/components/Dao/DaoLandingPage.tsx

```diff
@@ -28,30 +27,44 @@ export default class DaoLandingPage extends React.Component<IProps, null> {
   }
 
   private handleEditContent = () => {
-    showSimpleMessage(
-      {
-        title: "Edit Home Page",
-        body:
-          <>
-            <div>Editing the content on this DAO’s home page will soon be possible via proposal. Stay tuned!</div>
-            <div>For now, if you need a change made to a DAO’s home page content, please contact us at <a href="https://support@daostack.zendesk.com" target="_blank" rel="noopener noreferrer">support@daostack.zendesk.com</a></div>
-          </>,
-      }
-    );
-  }
+    showSimpleMessage({
+      title: "Edit Home Page",
+      body: (
+        <>
+          <div>
+            Editing the content on this DAO’s home page will soon be possible
+            via proposal. Stay tuned!
+          </div>
+          <div>
+            For now, if you need a change made to a DAO’s home page content,
+            please contact us at{" "}
+            <a
+              href="https://support@daostack.zendesk.com"
+              target="_blank"
+              rel="noopener noreferrer"
+            >
+              support@daostack.zendesk.com
+            </a>
+          </div>
+        </>
+      ),
+    });
+  };
 
   public render() {
     const daoState = this.props.daoState;
-    const customData = customDaoInfo[targetedNetwork()]?.[daoState.id.toLowerCase()];
+    const customData =
+      customDaoInfo[targetedNetwork()]?.[daoState.id.toLowerCase()];
 
     this.disqusConfig.url = `${process.env.BASE_URL}/dao/${this.props.daoState.address}/discussion`;
     this.disqusConfig.identifier = this.props.daoState.address;
     this.disqusConfig.title = "Discuss " + this.props.daoState.name;
 
     return (
       <div className={css.landingPage}>
-
-        <BreadcrumbsItem to={"/dao/" + daoState.address}>{daoState.name}</BreadcrumbsItem>
+        <BreadcrumbsItem to={"/dao/" + daoState.address}>
+          {daoState.name}
+        </BreadcrumbsItem>
```

Comment:
why this update ?

Reply:
my IDE did formatting of JS as I save file